libr/core/rtr.c

```diff
@@ -810,7 +814,10 @@ R_API int r_core_rtr_http(RCore *core, int launch, const char *path) {
 			eprintf ("TODO: Use different eval environ for scr. for the web\n");
 			eprintf ("TODO: Visual mode should be enabled on local\n");
 		} else {
-			HttpThread ht = { core, launch, path };
+			char *tpath = path + 1;
```

Comment:
Thanks - I'll use r_str_trim_const() since these are const char and I'm only looking to trim leading whitespace.